python/main.py

```diff
@@ -1,51 +1,44 @@
-from flask import (Flask, request, jsonify, Response)
+from flask import Flask, request, Response
 
 app = Flask(__name__)
 
 DATA_STORE = {}
 
-"""
-Serve up a welcome page at the root url
-"""
-@app.route('/', methods=['GET'])
-def homepage():
 
-    return Response('Hello, world. Looks like everything is working. \
-                     You got this!')
+@app.route("/", methods=["GET"])
+def home():
+    """Serve up a welcome page at the root URL."""
+    return Response("Hello, world. Looks like everything is working. \
+                     You got this!", mimetype="text/plain")
 
 
-"""
-Save key and value pairs from the query string
-   http://localhost:8080/set?key=value
-"""
-@app.route('/set/')
-def set():
+@app.route("/set/")
+def set_value():
+    """Save key and value pairs from the query string.
+
+       Try: http://localhost:8080/set?key=value
+    """
     # Loop through each key in the query string and add it to our dictionary
     for key, value in request.args.items():
         DATA_STORE[key] = value
 
         # Log the values to the debug console
-        print(f'recieved {key} with value of {value}')
+        print(f"recieved {key} with value of {value}")
 
     # Let the client know the operation was successful
-    return Response(f'saved {len(request.args)} value(s)')
-
+    return Response(f"saved {len(request.args)} value(s)", mimetype="text/plain")
 
-"""
-Return all saved keys to the client
-"""
-@app.route('/keys/')
-def keys():
-    if len(DATA_STORE) > 0:
-        all_values = [f'{key}={value}' for key, value in DATA_STORE.items()]
 
-        return Response((', ').join(all_values))
-    else:
-        return Response('No keys have been set.')
+@app.route("/keys/")
+def get_keys():
+    """Return all saved keys to the client.
 
+       Try: http://localhost:8080/keys/
+    """
+    if DATA_STORE:
```

Comment:
Simplified this conditional. You may have wanted it the old way to increase alignment between the two variants.